src/Symfony/Component/DependencyInjection/Compiler/AutowireExceptionPass.php

```diff
@@ -21,16 +21,19 @@
 class AutowireExceptionPass implements CompilerPassInterface
 {
     private $autowirePass;
+    private $inlineServicePass;
 
-    public function __construct(AutowirePass $autowirePass)
+    public function __construct(AutowirePass $autowirePass, InlineServiceDefinitionsPass $inlineServicePass)
     {
         $this->autowirePass = $autowirePass;
+        $this->inlineServicePass = $inlineServicePass;
```

Comment:
shouldn't we unset these two properties when process is over to remove a few circular refs?